src/plugins/intel_gpu/docs/gpu_plugin_ops_enabling.md

```diff
@@ -11,36 +11,38 @@
-        | [src/graph/scatter_elements_update.cpp](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_gpu/src/graph/scatter_elements_update.cpp) | Code for scatter_elements_update_inst.h |
-        | [primitives/scatter_elements_update.hpp](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_gpu/include/intel_gpu/primitives/scatter_elements_update.hpp) | GPU primitive definition |
+        | [group_normalization_impls.cpp](https://github.com/openvinotoolkit/openvino/blob/102ebddb3ce484cfc99142038b4dd1e48057b703/src/plugins/intel_gpu/src/graph/registry/group_normalization_impls.cpp#L20) | list up implemented kernels for kernel selection |
+        | [group_normalization_ref.cl](https://github.com/openvinotoolkit/openvino/blob/102ebddb3ce484cfc99142038b4dd1e48057b703/src/plugins/intel_gpu/src/graph/impls/ocl_v2/group_normalization_ref.cl) | OpenCL Kernel body. For more detail, please see [How to write OCL kernel](#writing-ocl-kernel) section |
+        | [group_normalization_ref.(cpp,hpp)](https://github.com/openvinotoolkit/openvino/blob/102ebddb3ce484cfc99142038b4dd1e48057b703/src/plugins/intel_gpu/src/graph/impls/ocl_v2/group_normalization_ref.cpp) | Counterpart of kernel body for host |
+        | [registry.hpp](https://github.com/openvinotoolkit/openvino/blob/102ebddb3ce484cfc99142038b4dd1e48057b703/src/plugins/intel_gpu/src/graph/registry/registry.hpp#L140) | Primitive registration. For example, registration for [group_normalization](https://github.com/openvinotoolkit/openvino/blob/102ebddb3ce484cfc99142038b4dd1e48057b703/src/plugins/intel_gpu/src/graph/registry/registry.hpp#L140).|
+        | [group_normalization_inst.h](https://github.com/openvinotoolkit/openvino/blob/102ebddb3ce484cfc99142038b4dd1e48057b703/src/plugins/intel_gpu/src/graph/include/group_normalization_inst.h) | Node type declaration for GPU program |
+        | [src/graph/group_normalization.cpp](https://github.com/openvinotoolkit/openvino/blob/102ebddb3ce484cfc99142038b4dd1e48057b703/src/plugins/intel_gpu/src/graph/group_normalization.cpp) | Code for group_normalization_inst.h |
+        | [primitives/group_normalization.hpp](https://github.com/openvinotoolkit/openvino/blob/102ebddb3ce484cfc99142038b4dd1e48057b703/src/plugins/intel_gpu/include/intel_gpu/primitives/group_normalization.hpp) | GPU primitive definition |
         | [common_types.h](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_gpu/src/kernel_selector/common_types.h) | Enum declaration for KernelType and arguments |
-
+        * Legacy infra for OCL kernels has seperate primitive registration for input specifications and kernel parameters. (e.g. [scatter_elements_update.cpp](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_gpu/src/graph/impls/ocl/scatter_elements_update.cpp))
+            But new infra `ocl_v2` approach does not need it.
```

Comment:
If not necessary, let's not mention `legacy infra`

Reply:
Removed this comment. Just left comment for a case of porting.